src/Components/HeroContainer/Hero.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
you can alos remove react imports if not used!

Reply:
Thank you removed